index.html

```diff
@@ -6,6 +6,392 @@
+			Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<span>4</span> Smartphones offer many additional features but impose a barrier of entry that seemed too high. Reliance on personal computers is also problematic, especially among disadvantaged patients. 
+			</p>
+			<p>
+			We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<span>5</span> Furthermore, adherence studies using text reminders show promising results.<span>6</span>  Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<span>7</span>
+			</p>
+			<p>
+			Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?
+			</p>
+		</article>
+
+		<!-- rotate the figure using css -->
+		<figure>
+			<img src="assets/phone.svg">
+		</figure>
+	</section>
+	<!-- end the question -->
+
+	<section class="row" id="meetFlorence">
+		<header>
+			<figure>
+				<img src="assets/florence.svg" alt="Florence is a friendly looking nurse with strawberry blonde hair">	
+			</figure>
+			<span class="subTitle">THE CONCEPT</span>
+			<h1>Meet Florence</h1>
+			<p>
+				Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen.  A chatbot at heart, patients would converse with her through text and picture messages. 
+			</p>
+			<p>
+				If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.
+			</p>
+		</header>
+
+		<!-- nest sections within the meetFlorence section -->
+		<!-- these sections will be styled as white boxes and share a common class -->
+
+		<section class="row underFlorence">
+
+		<!-- the firstHalf and secondHalf classes will both be on the same line, side-by-side -->
+
+			<section class="firstHalf">
+				<h1>SHE GETS TO KNOW YOU.</h1>
+				<p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+				<p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
+			</section>
+			<section class="secondHalf">
+				<div class="grayC">
+					Hi! I'm Florence. What's your name?
+				</div>
+				<div class="blueC">
+					Mike
+				</div>
+				<div class="grayC">
+					Nice to meet you, Mike. I can help you set up your account with a few simple questions.
+				</div>
+				<div class="grayC">
+					You can answer these now or whenever you like. I'm always here!
```

Comment:
Personally, I'd make then `<li>`(s) in an `<ol>`, since it is a list of statements.